### DISCUSSION TITLE
copilot is not working after payment

### DISCUSSION THREAD

**Discussion by brokeyourbike** [community/community#30665] [Copilot Conversations] [answered]
I have used copilot for months during the beta access I guess, and it was working fine. But after I have decided to pay for the service it's no longer working.

Tried rebooting machine, reinstated extension, tried Nightly version, and still seeing this freaking notification:
![image](https://user-images.githubusercontent.com/1554371/186428086-c146554f-9638-424b-8553-e9895310fe03.png)

Like what is going on? If you willing to charge people for the service, at least make this subscription thing work 

PS: there is a lot of similar discussions, but none of them proposed the solution, so I am opening this one to give more attention to the issue

**Comment by brokeyourbike** [comment, marked as answer]
Looks like I have fixed it.
Steps to follow on macOS:
- remove [VS Code](https://code.visualstudio.com/)
- `rm -rf ~/Library/Application\ Support/Code/`
- install [VS Code](https://code.visualstudio.com/)
- install [copilot](https://github.com/features/copilot)

**Comment by davecheney** [comment]
@brokeyourbike please upgrade vscode and copilot, the older versions of vscode do not permit you to upgrade beyond copilot 1.31

**Reply by Bortronx** [reply]
@davecheney
 copilot doesn't work with visual studio 2022 even though is advertised as working.

VS2022 Version 17.4.0 Preview 1.0


GitHub Copilot 1.42.0.0 (v1.42.0.0@5f1bd5ff5)
GitHub Copilot Agent v1.42.0
GitHub Copilot is globally enabled

No suggestions.
